rclpy/rclpy/node.py

```diff
@@ -141,7 +141,7 @@ def __init__(
         self.__guards: List[GuardCondition] = []
         self.__waitables: List[Waitable] = []
         self._default_callback_group = MutuallyExclusiveCallbackGroup()
-        self._parameters_callback = None
+        self._parameters_callback:Callable[List[Parameter],SetParametersResult]=[]
```

Comment:
Thanks for adding a type annotation, you should add some spaces to please the linters and make it more readable (similar to the other annotations above):

 ```suggestion
        self._parameters_callback: Callable[List[Parameter], SetParametersResult] = []
```

Reply:
If I change it in your way then I wont be able to add element into the self._parameters_callback it will instead override the self._parameters_callback as far I see it